packages/core-components/src/components/table/table.tsx

```diff
@@ -52,6 +62,7 @@ export class TableComponent {
 
   componentDidRender() {
     this.setCellSize();
+    this.setHeaderResize();
```

Comment:
should be conditional if resize is set to true, otherwise all columns are resizable all the time.